lib/features/digital_guide_view/tabs/surrounding/data/models/surrounding_response.dart

```diff
@@ -18,20 +107,67 @@ class SurroundingResponse with _$SurroundingResponse {
+  ) =>
       _$SurroundingResponseTranslationsFromJson(json);
 }
 
 @freezed
-class SurroundingResponseTranslation with _$SurroundingResponseTranslation {
-  const factory SurroundingResponseTranslation({
+class SurroundingResponseTranslationsDetails
+    with _$SurroundingResponseTranslationsDetails {
+  const factory SurroundingResponseTranslationsDetails({
+    @JsonKey(name: "are_no_barriers_comment")
+    required String areNoBarriersComment,
+    @JsonKey(name: "are_lowered_curbs_comment")
+    required String areLoweredCurbsComment,
+    @JsonKey(name: "is_pavement_rough_comment")
+    required String isPavementRoughComment,
     @JsonKey(name: "are_parking_spaces_comment")
-    required String parkingSpacesComment,
-  }) = _SurroundingResponseTranslation;
+    required String areParkingSpacesComment,
+    @JsonKey(name: "are_parking_spaces_for_pwd_comment")
+    required String areParkingSpacesForPwdComment,
+    @JsonKey(name: "are_high_curbs_at_parking_space_for_pwd_comment")
+    required String areHighCurbsAtParkingSpaceForPwdComment,
+    @JsonKey(name: "closest_buildings") required String closestBuildings,
+    @JsonKey(name: "is_path_to_accessible_entrance_marked_comment")
+    required String isPathToAccessibleEntranceMarkedComment,
+    @JsonKey(name: "are_dangerous_elements_comment")
+    required String areDangerousElementsComment,
+    @JsonKey(name: "is_lit_comment") required String isLitComment,
+    @JsonKey(name: "is_noisy_street_near_building_comment")
+    required String isNoisyStreetNearBuildingComment,
+    @JsonKey(name: "is_sidewalk_separated_from_the_road_comment")
+    required String isSidewalkSeparatedFromTheRoadComment,
+    @JsonKey(name: "accessibility_level_for_motor_disability_comment")
+    required String accessibilityLevelForMotorDisabilityComment,
+    @JsonKey(name: "accessibility_level_for_blind_comment")
+    required String accessibilityLevelForBlindComment,
+    @JsonKey(name: "accessibility_level_for_visually_impaired_comment")
+    required String accessibilityLevelForVisuallyImpairedComment,
+    @JsonKey(name: "accessibility_level_for_hard_of_hearing_comment")
+    required String accessibilityLevelForHardOfHearingComment,
+    @JsonKey(name: "accessibility_level_for_high_sensory_sensitivity_comment")
+    required String accessibilityLevelForHighSensorySensitivityComment,
+    @JsonKey(name: "accessibility_level_for_cognitive_difficulties_comment")
+    required String accessibilityLevelForCognitiveDifficultiesComment,
+    required String comment,
+  }) = _SurroundingResponseTranslationsDetails;
 
-  factory SurroundingResponseTranslation.fromJson(Map<String, dynamic> json) =>
-      _$SurroundingResponseTranslationFromJson(json);
+  factory SurroundingResponseTranslationsDetails.fromJson(
+    Map<String, dynamic> json,
+  ) =>
+      _$SurroundingResponseTranslationsDetailsFromJson(json);
+}
+
+bool _stringToBool(String value) {
+  return value == "True";
+}
```

Comment:
Maybe it'd be good to first parse string's value to lower (or upper) case in case someone clever'd change it on pwr's site :)